pkgs/development/python-modules/pycdio/default.nix

```diff
@@ -4,14 +4,17 @@
   buildPythonPackage,
   fetchPypi,
   setuptools,
-  nose,
-  pkgs,
+  pkg-config,
+  swig,
+  libcdio,
+  libiconv,
+  pytestCheckHook,
 }:
 
 buildPythonPackage rec {
   pname = "pycdio";
   version = "2.1.1";
```

Comment:
We should probably bump to the current Git HEAD, which contains the `substituteInPlace` and another test fix.